### PR TITLE
fix: make RecentlyViewedArtworks connection work

### PR DESCRIPTION
**Pull request by olerichter00**
Prepares MP for [CX-3411]

## Description

Make RecentlyViewedArtworks connection work. It wasn't working before :/

[CX-3411]: https://artsyproduct.atlassian.net/browse/CX-3411?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/schema/v2/me/recentlyViewedArtworks.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    const totalPages = Math.ceil(totalCount / size)
+
+    return {
+      totalCount,
+      pageCursors: createPageCursors({ ...args, page, size }, totalCount),
+      ...connection,
+      pageInfo: {
+        ...connection.pageInfo,
+        hasPreviousPage: page > 1,
+        hasNextPage: page < totalPages,
```
Comment (olerichter00) [inline src/schema/v2/me/recentlyViewedArtworks.ts:48]: Overriding `pageInfo` here because `connectionFromArraySlice` doesn't work correctly :/

**Comment by lordkiz** [comment]
✅

File: src/schema/v2/me/index.ts [hunk 2]
```diff
@@ -53,7 +53,7 @@ import { MyCollectionInfo } from "./myCollectionInfo"
 import { myLocationType } from "./myLocation"
 import { NewWorksByInterestingArtists } from "./newWorksByInterestingArtists"
 import { ManagedPartners } from "./partners"
-import { RecentlyViewedArtworks } from "./recently_viewed_artworks"
```
Comment (MounirDhahri) [inline src/schema/v2/me/index.ts:56]: **thought:** is the previous file no longer used? if so maybe we can remove it?

Comment (MounirDhahri) [reply src/schema/v2/me/index.ts:56]: my bad, I misssed it. thanks Ole

**Comment by mzikherman** [comment]
Not to be a nit, but this _seems_ to have been working fine on Force (as a rail) for quite a while. Could you describe what wasn't working (if you recall)? I'm wondering if this PR _then_ somehow necessitated this PR - https://github.com/artsy/metaphysics/pull/4867 (which again, I believe was working fine for a quite a while), which itself may have led to some buggy behavior, etc.

**Pull request merged**